reframe/core/config.py

```diff
@@ -440,6 +439,31 @@ def select_subconfig(self, system_fullname=None,
         self._subconfigs[system_fullname] = local_config
 
 
+def find_config_files(config_path=None, config_file=None):
+    res = []
+    if config_path:
+        for p in config_path:
+            if os.path.exists(p + '/settings.py'):
+                res.append(p + '/settings.py')
+            elif os.path.exists(p + '/settings.json'):
+                res.append(p + '/settings.json')
+            else:
+                getlogger().debug('Loading the generic configuration')
```

Comment:
```suggestion
        for p in config_path:
            py_config = os.path.join(p, 'settings.py')
            json_config = os.path.join(p, 'settings.json')
            if os.path.exists(py_config):
                res.append(py_config)
            elif os.path.exists(json_config):
                res.append(json_config)
```

I think this debug message makes more sense to be emitted from the actual loader, that will eventually fall back to the builtin config.

Reply:
This is actually an unfortunate copy-paste 😛 I wanted to print a message that says that this path will be omitted. I changed it, let me know if you think we should skip it

Reply:
I think we could omit the log message here altogether as there are different places where we can see which configuration files were picked.

Reply:
I am worried that users may not notice this the directory needs to have a `settings.py` or json file in the directory. As it is now with -vv I am getting this output:
```bash
Loading user configuration
No 'settings.py' or 'settings.json' found in 'wrong_path', path will be ignored
Loading the generic configuration
Detecting system using method: 'hostname'
...
```
Without the message `No 'settings.py' or 'settings.json' found in 'wrong_path', path will be ignored` I think it can be hard to understand what you are doing wrong.

Reply:
Ok, I agree, it  helps.